### PR TITLE
KG - Research Master ID Integration Refactor

### PR DESCRIPTION
**Pull request by kayla-glick**

### CODE REVIEW

File: app/models/protocol.rb [hunk 1]
```diff
@@ -261,6 +255,10 @@ def is_project?
     self.type == 'Project'
   end
 
+  def has_human_subject_info?
+    research_types_info.human_subjects
```
Comment (ghost) [inline app/models/protocol.rb:259]: This should return a Boolean. I think appending `.any?` should work.

**Comment by wtholt** [comment]
The functionality for this integration isn't really nailed down yet. I think we should close this until we get a better idea of how this feature will work in the long run. What do you think @jleonardw9 ?

File: app/models/protocol.rb (hunk 1 above)
Comment (kayla-glick) [reply app/models/protocol.rb:259]: @jay
human_subjects is a boolean attribute on research_types_info

**Comment by jleonardw9** [comment]
@wtholt Lets have a quick meeting tomorrow and discuss.

**Comment by jleonardw9** [comment]
Kyle, we're going to close this for now. When all the functionality is worked out then we'll pull this in.

**Pull request closed without merging**